platforms-wg/glossary/_index.md

```diff
@@ -1,43 +1,43 @@
-“DevOps is a methodology in which teams own the entire process from application development to production operations.”<sup><a href="https://glossary.cncf.io/devops/">2</a></sup> While DevOps practices can be implemented by teams without developing a dedicated platform, it can be useful to view platform engineering as an approach to scaling DevOps principles through the delivery and management of a unified platform that serves the entire organization. This shared platform aims to streamline the development, deployment, and operational processes, providing a standardized and efficient environment for software delivery. Although DevOps and platform engineering converge on the objectives of optimizing software delivery and operational performance, platform engineering distinctively focuses on the development of a tangible product—the platform itself—to facilitate these goals.
+"DevOps es una metodología en la que los equipos son dueños de todo el proceso, desde el desarrollo de aplicaciones hasta las operaciones de producción."<sup><a href="https://glossary.cncf.io/devops/">2</a></sup> Aunque las prácticas de DevOps pueden ser implementadas por equipos sin desarrollar una plataforma dedicada, puede ser útil ver la ingeniería de plataforma como un enfoque para escalar los principios de DevOps a través de la entrega y gestión de una plataforma unificada que sirva a toda la organización. Esta plataforma compartida tiene como objetivo racionalizar los procesos de desarrollo, despliegue y operativa, proporcionando un entorno estandarizado y eficiente para la entrega de software. Aunque DevOps y Platform Engineering convergen en los objetivos de optimizar la entrega de software y el rendimiento operativo, Platform Engineering se centra de manera distintiva en el desarrollo de un producto tangible: la plataforma en sí misma, para facilitar estos objetivos.
 
-## Platform users
-The people who directly use capabilities of the platform, including but not limited to app developers, app operators, data scientists, commercial off-the-shelf (COTS) software operators, and information workers — whoever runs software on the platform, uses platform-provided capabilities, or requires insights into platform use. Platform users may include other platform engineers creating higher level platform services on top of lower level capabilities.
+## Usuarios de la Plataforma
+Las personas que utilizan directamente las capacidades de la plataforma, incluyendo pero no limitado a desarrolladores de aplicaciones, operadores de aplicaciones, científicos de datos, operadores de software comercial off-the-shelf (COTS) y trabajadores de la información - quienes ejecutan software en la plataforma, utilizan capacidades proporcionadas por la plataforma o requieren información sobre el uso de la plataforma. Los usuarios de la plataforma pueden incluir a otros ingenieros de plataforma que crean servicios de plataforma de un nivel superior sobre capacidades de niveles inferiores.
```

Comment:
whats the meaning of "information workers"? "Trabajadores de la información" seems a little bit off for me if I'm honest :)